src/View/User/_addUserForm.html.twig

```diff
@@ -0,0 +1,43 @@
+{% block stylesheet %}
```

Comment:
since this file is a component to include, please shift the stylesheet inside the page supposed to include this file